test_sets/smoke_tests.txt

```diff
@@ -58,16 +60,16 @@
 ./test_scripts/Smoke/Registration/004_Reregister_App.lua
 ./test_scripts/Smoke/Registration/005_Reregister_App_after_disconnect.lua
 ./test_scripts/Smoke/Registration/006_Reregister_App_if_two_apps_are_registered.lua
-./test_scripts/Smoke/Resumption/001_Resumption_3rd_ignition_cycle.lua
+./test_scripts/Smoke/Resumption/001_Resumption_3rd_ignition_cycle.lua 2976
```

Comment:
Well, this file has no defined syntax. It is used by script runner to execute script in every line. 
CI (Jenkins) use the second word (after the path to script) to specify in the report if this failed due to known reason or no. 
We can use it as a line above, but it will make parsing a little bit more complicated. 
Also, we have already used such practice in existing sets :+1: https://github.com/smartdevicelink/sdl_atf_test_scripts/blob/develop/test_sets/policies_happy_paths_HTTP.txt#L33

Can you specify the reason why do you see the line above is a better solution then link after the script?

Reply:
@jacobkeeler Please notice mentioned https://github.com/smartdevicelink/sdl_core/issues/2976 issue reproduced only sometimes

Reply:
Hmmm, seems that I can only reproduce this issue if Bluetooth is enabled

Reply:
@LuxoftAKutsan Really, if this is used by a separate script runner, then that runner should be provided to define this format.

Reply:
@jacobkeeler , CI(Jenkins) has its own code for a parsing of test sets and issues from it. We have already some version of the script runner in https://github.com/smartdevicelink/sdl_atf/tree/feature/script_runner according to the proposal https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0128-atf-script-runner.md, so we can add a logic from CI in script runner  in (https://github.com/smartdevicelink/sdl_atf/tree/feature/script_runner) and CI will use script runner without additional code.

Reply:
@jacobkeeler, added in https://github.com/smartdevicelink/sdl_atf/commit/2a014598a4f94b666bbe46a819c66e278b4a4660